database/src/main/resources/db/migration/V1_50__external_interfaces_description.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TYPE interface_data_content AS ENUM (
```

Comment:
number 50 migration is already in master. So rename migration file.

Reply:
There is a lot of stuff to fix in addition to this, I was aware of this problem also. I'm in progress fixing the problems. If I could cancel the review request, that would be nice feature by GitHub :)